src/test/java/peoplesoft/model/person/UniquePersonListSerdesTest.java

```diff
@@ -107,6 +107,7 @@ public void deserialize_invalidPersonElement_throwsJsonMappingException() {
                 BENSON.getPhone().toString(),
                 BENSON.getAddress().toString(),
                 BENSON.getEmail().toString(),
+                BENSON.getRate().getAmount().printFullValue(), // TODO
```

Comment:
Is it the make it more user friendly thing? $5 / hour rather than the 5/H

Reply:
Currently `printFullValue()` is the correct format for the JSON serialization so no need to be concerned. Just a marker there in case there is anything to change for `Rate` in the future